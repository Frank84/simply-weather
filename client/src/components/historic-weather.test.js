import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import HistoricWeather from './historic-weather';

it('renders properly', () => {
  const wrapper = shallow(<HistoricWeather />);
  expect(toJson(wrapper)).toMatchSnapshot();
});