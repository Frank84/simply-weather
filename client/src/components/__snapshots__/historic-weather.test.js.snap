// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<Fragment>
  <canvas
    height="100"
    id="myChart"
    style={
      Object {
        "margin": "1rem auto",
        "maxWidth": "400px",
        "width": "100%",
      }
    }
  />
</Fragment>
`;
